.github/workflows/python_package.yml

```diff
@@ -118,6 +121,35 @@ jobs:
             compiler: gcc
             method: wheel
             python_version: '3.10'
+          - os: ubuntu-latest
+            container: lightgbm.azurecr.io/vsts-agent:manylinux_2_28_x86_64
+            task: mpi
+            compiler: gcc
+            method: source
+            python_version: '3.10'
+            in-ubuntu-base-container: 'false'
+            setup-conda: 'false'
+          - os: ubuntu-latest
+            container: &ubuntu_latest_image mcr.microsoft.com/mirror/docker/library/ubuntu:22.04
```

Comment:
Why not simply 'ubuntu:22.04'?..

Like here:
https://github.com/microsoft/LightGBM/blob/9124af3e61bc7deae4bf7538c8a872b11f4e3a2c/.github/workflows/cpp.yml#L42